pkg/resources/stream.go

```diff
@@ -243,17 +243,20 @@ func DeleteStream(d *schema.ResourceData, meta interface{}) error {
 
 	err = snowflake.Exec(db, q)
 	if err != nil {
-		return errors.Wrapf(err, "error deleting stream %v", d.Id())
+		return errors.Wrapf(err, "error deleting stream %v", data.Id())
 	}
 
-	d.SetId("")
+	data.SetId("")
 
 	return nil
 }
 
 // UpdateStream implements schema.UpdateFunc
-func UpdateStream(d *schema.ResourceData, meta interface{}) error {
-	streamID, err := streamIDFromString(d.Id())
+func UpdateStream(data *schema.ResourceData, meta interface{}) error {
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>R014: *schema.ResourceData parameter of CreateFunc, ReadFunc, UpdateFunc, or DeleteFunc should be named d